Testsuite/test/run_testsuite_with_cmake

```diff
@@ -154,7 +154,11 @@ test_directory()
       START=`date +%s`
       run_local_cgal_test &
 
-      if wait_for_process "$!" "1200" "5"
+      TIMEOUT=1200
+      if [ "$1" = "Polyhedron_Demo" ]; then
+        TIMEOUT=5400
+      fi
+      if wait_for_process "$!" "$TIMEOUT" "5"
```

Comment:
By the way, the name `TIMEOUT` is not right, because that is the timeout divided by 5.